wiki/Beatmapping/Health/en.md

```diff
@@ -0,0 +1,101 @@
+---
+tags:
+- HP
+- Health Drain
+- HP Drain
+- Passive Drain
+- Drain
+---
+
+# Health
+
+Health or more commonly known as *HP*, is the value that decides if a player has failed or passed a [beatmap](/wiki/Beatmaps). Health can be seen in the health bar which updates as a beatmap progresses. Health is largely determined by the HP difficulty setting, which is set by the mapper of a beatmap.
+
+The HP difficulty settting controls how heavily a player is penalized for missing notes and how much health is gained back by accurately hitting [hit objects](/wiki/Hit_Objects). HP values range from HP0 to HP10, where HP10 is the highest and most punishing setting and HP0 the most forgiving. On HP10 the health drains rapidly and a single miss may be enough to fail a player, whereas on HP0 multiple combos can be missed before failing a player. The HP difficulty setting can be changed alongside other difficulty settings in the [Beatmap Editor](/wiki/Beatmap_Editor). Doing this will change a Ranked beatmap's status to unranked and will erase all local scores on it.
```

Comment:
> Doing this will change a Ranked beatmap's status to unranked and will erase all local scores on it.

Wouldn't changing anything in the beatmap and saving it locally erase all local scores for that beatmap? If so, it seems strange to me for why this article says this. Maybe put it in the beatmap editor article instead or make sure that all articles relating to the beatmap editor/song settings also has this too?

Reply:
agree, ill remove for now and make a reminder to put it in the beatmap editor article when i get to that